client/.eslintrc.json

```diff
@@ -22,5 +23,6 @@
         "react"
     ],
     "rules": {
+        "no-console": "off"
```

Comment:
I'm not sure we want this rule because we do want to be notified on unnecessary console statements. They let out info that we don't necessarily want to expose.